src/getCoinCombination.test.js

```diff
@@ -7,4 +7,25 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it(`throw an error if 'cents' is negative non-integer number`, () => {
```

Comment:
The test description should be more precise. Consider rephrasing it to something like: 'should throw an error if cents is a negative non-integer number'.